### ISSUE TITLE
Single style families check?

### ISSUE THREAD

**Issue by davelab6** [googlefonts/fontbakery#2798]
@felipesanches Do we have a check for single style families that are not Regular? 

Eg, can we check for the situation in https://github.com/ManiackersDesign/monomaniac/issues/2 ?

**Comment by felipesanches** [comment]
yes, we do have **com.google.fonts/check/metadata/has_regular**
`METADATA.pb: According Google Fonts standards, families should have a Regular style.`

https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/has_regular

**Comment by m4rc1e** [comment]
The check you've linked to ensures that every family has a Regular weight. I think Dave wants a check to ensure that single weight families only have a Regular style. I've written about these families in the [doc](https://gist.github.com/m4rc1e/8f4c4498519e8a36cd54e16a004275cb#single-weight-families)

**Comment by felipesanches** [comment]
yes, I understand that. My understanding is that the **has_regular** check indirectly detects absolutely all instances of this kind of problem since all single-style families lacking a regular do indeed lack a regular.

**Comment by felipesanches** [comment]
please reopen if needed, @davelab6

**Comment by m4rc1e** [comment]
I know what Dave is asking for so I'll explain.

For single weight families which don't have the appearance of a Regular, we want to append "One" to the family name e.g [Patua One](https://fonts.google.com/specimen/Patua+One). This allows us to keep the family name in case we want to expand the family. If we don't do this we lose the family name for future expansions and have to end up appending "2" to the family name e.g https://fonts.google.com/?query=lobster.

**Comment by felipesanches** [comment]
I'll try to address this one before 0.7.24

**Comment by m4rc1e** [comment]
Thank you!

**Comment by felipesanches** [comment]
> I'll try to address this one before 0.7.24

I wish I had not said that, more than an year ago :-P

Now that I came back here, I think I understand the problem. I suspect there's no easy way to auto-detect what subjectively "looks like regular".
